blog-post.html

```diff
@@ -1,12 +1,125 @@
-<!DOCTYPE html>
+<!DOCTYPE HTML>
 <html>
   <head>
-    <title>Blog Post</title>
+    <title>Learn More</title>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link href="blog-post.css" rel="stylesheet">
   </head>
   <body>
-    <section id="header"></section>
-    <section id="main"></section>
+    <a href="index.html" style="color: darkcyan">T*H*I*S*<span style="color: brown">L*I*N*K*S</span><span style="color: darkorange">*H*O*M*E</span></a>

```

Comment:
One minor hangup I noticed was the "return home button" jammed up in the top left corner of the pages, You could move this anchor tag to wrap your page headers so that the headers themselves are clickable. This is standard across most websites you will find online today. It will also make UX a bit easier for anyone who were to "visit" your site.